packages/export-framework/start-server.js

```diff
@@ -9,6 +9,8 @@ const CoreConfig = require('./config/core');
 const CustomConfig = require('./config/custom');
 
 if (!process.env.LIVERELOAD_PORT) process.env.LIVERELOAD_PORT = 5010;
+if (!process.env.LIVERELOAD_HOST) process.env.LIVERELOAD_HOST = 'localhost';
```

Comment:
Can we change all refs to the live reload host to use the exposed host env instead? Live reload should simply use the same host as the service.